FWCore/Integration/test/ESTestAnalyzers.cc

```diff
@@ -157,9 +158,25 @@ namespace edmtest {
                                            << ": Data values = " << dataA->value() << "  " << dataZ->value();
     }
   }
+
+  class ESTestAnalyzerJ : public edm::EDAnalyzer {
```

Comment:
Why change `ESTestAnalyzerA` to be a stream module but then make the new `ESTestAnalyzerJ` a legacy?

Reply:
I made ESTestAnalyzerJ a stream module.